### ISSUE TITLE
Exit Code 101 when Attempting to Start Joshuto From `sxkhd` Binding

### ISSUE THREAD

**Issue by kevindirect** [kamiyaa/joshuto#286]
I switched my sxhkd keybinding to open a file manager from `lf` to `joshuto`. The sxhkd keybind looks like this:
```
super + f
	$TERMINAL -e $FILE
```
where `TERMINAL=st` and `FILE=joshuto`.

The `joshuto` process quits with exit code 101 when launched this way, the result is the same when this command (`$TERMINAL -e $FILE`) is run from the terminal, though `joshuto` itself works when run from the shell (`zsh`).

I'm not too familiar with rust but I assume [101 means a panic](https://rust-cli.github.io/book/in-depth/exit-code.html). This keybind worked fine with `ranger` and `lf`. Using literals instead of environment variables, or removing the `-e`, doesn't change the result.

**Comment by kamiyaa** [comment]
If what you're saying is `joshuto` immediately quits upon startup, then there is a bug somewhere.

If you're saying upon exit, `joshuto` exits with 101, then its probably because you have exit to directory set up.
See here: https://github.com/kamiyaa/joshuto/blob/main/docs/configuration/keymap.toml.md#quit-quit-joshuto

**Comment by kevindirect** [comment]
Thanks for the prompt response!

If I'm understanding right, the second case happens during a user-triggered quit. No, joshuto quits by itself (no opportunity for user input). I get this when I run `st -e joshuto` from my terminal:
> child exited with status 101

`joshuto` by itself from the terminal works fine.

Also something important I forgot to mention that I remember now. After a restart `xst -e joshuto` works, a joshuto instance is opened properly. After that joshuto instance is closed a new one can't be opened and I get the above 101 error code.

**Comment by kevindirect** [comment]
I set joshuto to a new keybinding. I noticed new instances spawn fine as long as there is a window already open. On an empty workspace, pressing the keybind once doesn't work to spawn. I have to press it multiple times (spam it a little) to open a window. I think joshuto is spawning and immediately quitting. If pressed fast enough, a new instance can spawn "off of" the first instance that quits. Once the workspace has something in it, new joshuto windows can be spawned easily.

All of this might have something to do with my window manager and hotkey daemon (bspwm, sxhkd). I might try a different window manager and see if I still have this problem. I'll report it here if I try that.
